src/routes.jsx

```diff
@@ -1,17 +1,27 @@
 import React from 'react';
-import { Route, IndexRoute } from 'react-router';
+import { BrowserRouter, Route, Switch} from 'react-router-dom';
 
-import App from './components/App';
+import NavigationBar from './components/NavigationBar';
+import './styles/App.css';
 import SearchPage from './containers/SearchPage';
 import AboutPage from './components/AboutPage';
 import ContributorsPage from './components/ContributorsPage';
 import NotFoundPage from './components/NotFoundPage';
 
-export default (
-  <Route path="/" component={App}>
-    <IndexRoute component={SearchPage}/>
-    <Route path="about" component={AboutPage}/>
-    <Route path="contributors" component={ContributorsPage}/>
-    <Route path="*" component={NotFoundPage}/>
-  </Route>
-);
+const Routes = () => {
+  return(
+    <BrowserRouter>
+      <div>
```

Comment:
I'd replace this div with the `App` component. That component will format the UI, add the navbar, and render all children passed to it, so the switch and routes will still work!